cmd/tests/test_state.go

```diff
@@ -49,7 +49,7 @@ func NewGlobalTestState(t *testing.T) *GlobalTestState {
 	logger := logrus.New()
 	logger.SetLevel(logrus.InfoLevel)
 	logger.Out = testutils.NewTestOutput(t)
-	hook := &testutils.SimpleLogrusHook{HookedLevels: logrus.AllLevels}
+	hook := testutils.NewLogHook()
```

Comment:
https://github.com/grafana/xk6-browser/issues/823

Reply:
Resolved by the latest force-push.